packages/paste-core/components/user-dialog/stories/index.stories.tsx

```diff
@@ -61,8 +61,9 @@ export const BasicUserDialog: StoryFn = () => {
             selected={selected === id2}
             onSelect={() => setSelected(id2)}
             data-testid="SECOND_ITEM"
+            href="https://www.google.com"
           >
-            <UserIcon decorative color="colorTextIcon" />
+            <UserIcon decorative color="inherit" />
```

Comment:
`currentColor` is the default, which is like `inherit`. Can omit the color prop here altogether